packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -280,8 +280,7 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, Lockable {
         // Sanity check that pool has enough balance to cover relay amount + proposer reward. Reward amount will be
         // paid on settlement after the OptimisticOracle price request has passed the challenge period.
         require(
-            l1Token.balanceOf(address(this)) >= depositData.amount + proposerBond &&
-                liquidReserves >= depositData.amount + proposerBond,
+            l1Token.balanceOf(address(this)) - bonds >= depositData.amount && liquidReserves >= depositData.amount,
```

Comment:
I'm not sure of a case to make _only_ the first part of this check fail, but maybe @chrismaree has ideas. This would be a good test to add. Gonna go ahead and merge because I think we should get this in, but we should definitely add this.